transformer_engine/pytorch/module/base.py

```diff
@@ -57,6 +57,12 @@
 _NUM_MAX_UB_STREAMS = 3
 _MIN_STREAM_PRIORITY, _MAX_STREAM_PRIORITY = None, None
 layers_atomic_ring_exchange = []
+_QUANTIZED_WEIGHT_TENSOR_TYPES = (
```

Comment:
We recently merged the change to introduce `QuantizedTensorBase` class so you should be able to remove this.